README.md

```diff
@@ -1,2 +1,132 @@
 # VR-Flight-Simulator
 Srishti 2019
+
+### PROJECT
+![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Game.jpeg)
+
+## ABSTRACT
+
+The project we made is known as VR Flight Simulator.It is a simulation of real life plane experience.
+In this project We had used a ultrasonic sensor, a bump sensor, a potentiometer sensor, a gyroscope
+sensor (esp8266), and a esp9250 module to make a joystick and a throttle handle. The joytick physically
+moved in vertical circular motion in left write direction, and it moved front and backwards producing 
+respective motion in game. The throttles went front and backwards to accelerate and decelerate the plane
+respectively.A game on pc was developed by us which contains a plane and some objects which can be shot.
+
+## MOTIVATION
+
+Our motivation for doing this project was the creativity of the idea and our fascination with the end result.
+
+## MECHANICAL ASPECT OF DESIGN
+
+ Project Mechanical Design
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Project%20Mechanical%20design.jpeg)
+ 
+  Plane Mesh Made From Wire
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/plane.jpeg)
+
+Our mechanical parts consists of:
+	
+ 1. **Joystick**	-This device is used for controlling the aeroplane so that user experiences reality ( gives us a real experience ), with the help of this device we tried to make environment like that of aeroplane .
+2. **Throttle** 	-This device is basically used for controling the speed of aeroplane.
+
+Both the devices are made from basic components such as ply , a man made springs , steel channels , ball bearings , and joining components
+
+## ELECTRONIC ASPECT OF DESIGN
+
+Electronics Consist of use of:
+
+1. **Ultrasonic sensor**	-for giving pitch to plane
+2. **Gyro sensor**	-for detecting right-left motion of plane
+3. **Potentiometer**	-for detecting throttle
+4. **Bump Sensor**	-for detecting Rocket shooting
+5. **Esp module**	-for receiving data from sensors and sending it to unity	
+
+## WORKING
+
+ While Playing
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/while%20playing.jpeg)
+
+* Taking input from 4 sensors(in form of nos.) and sending this input to unity software through esp module
+* After receiving data we are scaling it in unity to make it in suitable form and sending it to our game
+* Then we are using a software called trinus vr to project computer screen onto our mobile in form of 3d environment by making pc as server and mobile as receiver
+* This is it. Syncing sensor input and input in unity was very crucial.
+ 
+## COST STRUCTURE
+
+1. **Mechanical** 
+
+	|  Component |Cost  RS. |
```

Comment:
1. Write Cost(INR) in place of the Cost Rs. in both the table and make a single table.
2. Add total row in the table also.